data/siggraph-2019/10.1145-3306346.3322941.json

```diff
@@ -0,0 +1,98 @@
+	    "Neural networks"
+        ],
+
+        "Topic {Rendering, Animation and Simulation, Geometry, Images, Virtual Reality, Fabrication}": "Geometry",
+
+        "Co-authors from academia (boolean)": true,
+
+        "Co-authors from industry (boolean)": false,
+
+        "ACM Open Access (boolean)": false,
+
+        "PDF on the authors' webpage / institution (boolean)": true,
+
+        "PDF on Arxiv or any openarchive initiatives (boolean)": false,
+        "Arxiv/OAI page URL": "",
+
+        "PDF URL": "https://drive.google.com/file/d/1CJCM6EQyeUWwxdk6tl8cVxEIhV7s3DoA/view",
+        
+        "Project URL": "",
+
+        "Code available (boolean)": true,
+        "Code URL": "https://github.com/brownvc/planit",
+        "Code URL2": "",
+        "MD5 sum (for archives)": "",
+        "MD5 sum (for archives) URL2": "",
+        "git/hg/svn commit hash or revision number": "b42bfda1f7cd7c0e07874728ae7b809c09a3e6b7",
+        "git/hg/svn commit hash or revision number URL2": "",
+
+        "Software Heritage permalink": "https://archive.softwareheritage.org/swh:1:dir:5beb70169b7a0396c8e4bfb339d4994f6350120b;origin=https://github.com/brownvc/planit;visit=swh:1:snp:a3c298ae1199d97b23b00949037c588aef43d5cd;anchor=swh:1:rev:b42bfda1f7cd7c0e07874728ae7b809c09a3e6b7;path=//",
+       
+
+        "If code not available, pseudo-code available (boolean)": false,
+        
+        "If pseudo-code, could the paper be trivially implemented? {0..4}": 0,
+
+        "Software type {Code, Binary, Partial Code}": "Code",
+
+        "Software language": "python",
+        
+        "Code License (if any)": "All rights reserved to Brown",
+
+        "Are the code authors explicit? (boolean)": false,
+
+        "Build/Configure mechanism": "NA",
+
+        "Dependencies": "Not listed",
+        
+        "Does the software require paywall/proprietary software/material (boolean)?": false,
+
+        "Does the code need data (not examples) (boolean)": true,
+        
+        "Nature of the data (pretrained model, LUT...)": "pretrained model or training dataset",
+
+        "License of the data": "Data not provided.",
+
+        "Able to perform a replicability test (boolean)": false,
+
+        "If not able to perform a test, was it due to missing hardware/software? (boolean)": false,
+
+        "Documentation score {0=NA,1,2,3}": 0,
+
+        "Dependencies score {0=NA, 1,2,3,4,5}": 0,
+
+        "Build/configure score {0=NA, 1,2,3,4,5}": 0,
+
+        "Fixing bugs score (if any) {0=NA, 1,2,3,4,5}": 0,
+
+        "Replicate paper results score {0=NA, 1,2,3,4,5}": 0,
+
+        "Adaptability score to other contexts {0=NA, 1,2,3,4,5}": 0,
+
+        "Time spent for the test (code download to first successful run, [0,10], 10min slots, 100min max)": 1,
+
+        "Build instructions/comments": "Could not run due to missing data: the dataset is no longer available, and no pretrained model is given. The necessary packages are not listed in the readme file.",
```

Comment:
have you listed the deps you've had to add ?
(maybe hard to say as your system may already have some of them)